### PR TITLE
Alpweiden Herdenschutzhunde

### PR DESCRIPTION
**Pull request by rebert**
[try](https://mf-chsdi3.dev.bgdi.ch/ltrea_hunde/shorten/72f8579e15)

### CODE REVIEW

File: chsdi/models/vector/bafu.py [hunk 1]
```diff
@@ -12,6 +12,26 @@
 Base = bases['bafu']
 
 
+class AlpweidenHerdenschutzhunde(Base, Vector):
+    __tablename__ = 'alpenweiden_herdenschutzhunde'
+    __table_args__ = ({'schema': 'fauna', 'autoload': False})
+    __bodId__ = 'ch.bafu.alpweiden-herdenschutzhunde'
+    __queryable_attributes__ = ['name', 'kotname', 'kottel', 'hotemail', 'code_refverhalten', 'code_hundepraesenz', 'code_hinweis']
```
Comment (ltclm) [inline chsdi/models/vector/bafu.py]: * ``kotname`` does not exist in ``bafu_master.fauna.alpenweiden_herdenschutzhunde``, there is a column called ``kontname`` which is empty??
* ``kottel`` same as ``kotname``
* ``hotemail`` does not exist neither there is a column ``kontemail``

please fix the typos names of the columns

* ``code_refverhalten``, ``code_hundepraesenz``, ``code_hinweis`` are coded values, i would recommend to not implement attribute search on such attributes, furthermore 102 of 103 objects have the same value, in my eyes it's nonsense to have a attribute search here, @mariokeusen your opinion please?

Comment (ltclm) [inline chsdi/models/vector/bafu.py]: and please add db indexes on the queryable attributes.

File: chsdi/locale/empty_chsdi.po [hunk 2]
```diff
@@ -322,6 +322,51 @@ msgstr ""
 msgid "ch.swisstopo.digitales-hoehenmodell_25.metadata"
```
Comment (rebert) [inline chsdi/locale/empty_chsdi.po]: no more empty po this needs to be removed

**Comment by rebert** [comment]
https://github.com/geoadmin/service-sphinxsearch/pull/329

**Comment by rebert** [comment]
jenkins test

**Comment by rebert** [comment]
Verified and approved by @loicgasser

**Pull request merged**